backend/src/app.controller.ts

```diff
@@ -36,4 +38,10 @@ export class AppController {
   getUser(@Req() ID: string): Promise<User | null> {
     return this.userService.getUser(ID);
   }
+
+  // tag/search/?q=
+  @Get('tag/search/')
```

Comment:
This request handler should only be temporary, it will be replaced with appropriate handler in /requests